koji-setup/deploy-mash.sh

```diff
@@ -46,19 +46,19 @@ EOF
 mkdir -p "$MASH_SCRIPT_DIR"
 cp -f "$SCRIPT_DIR"/mash.sh "$MASH_SCRIPT_DIR"
 mkdir -p /etc/systemd/system
-cat > /etc/systemd/system/mash.service <<- EOF
+cat > /etc/systemd/system/mash@$TAG_NAME.service <<- EOF
```

Comment:
I think for a case like this, we want the service file on disk to be named `mash@.service` to indicate to systemd that we want to pass a parameter, but not hard-code the parameter for all instantiations of the service.